issue_07/script.js

```diff
@@ -0,0 +1,32 @@
+const getData = new Promise((resolve) => {
+    const lists = [{
+        img: "1.png",
+        alt: "画像1",
+        text: "ブックマーク"
+    }, {
+        img: "2.png",
+        alt: "画像2",
+        text: "メッセージ"
+    }];
+    setTimeout(() => {
+        resolve(lists);
+    }, 3000);
+});
+const loadingImg = document.getElementById('js-loadingImg');
+const ul = document.getElementById('js-ul');
+getData.then((value) => {
+    const fragment = document.createDocumentFragment();
+    for (const item of value) {
+        const li = document.createElement('li');
+        const p = document.createElement('p');
+        const img = document.createElement('img');
+
+        p.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(p).appendChild(img);
+    }
+    loadingImg.remove();
+    ul.appendChild(fragment);
```

Comment:
IMO
I thought it would be easier to read if the part that builds the DOM was made into a function.

Reply:
Thank you for pointing this out!
I fixed it so that it is grouped into a function and executed when the value is resolved.
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/47afdb7aab718f32134a19069a827ca4bcb05276
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/01581850e7e22af0dd51f2a8dfc6407c5203c8f7